scripts/regen-goldens.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# List of sub‑projects and their tests
+projects=(
+  "plutus-conformance"
+  "plutus-tx-plugin"
+  "plutus-ledger-api"
+  "plutus-benchmark"
+  "cardano-constitution"
+  "plutus-tx"
+  "plutus-core"
+)
+
+tests_plutus_conformance=(
+  "cabal run haskell-conformance -- --accept"
+  "cabal run haskell-steppable-conformance -- --accept"
+  "cabal run agda-conformance -- --accept"
+)
+tests_plutus_tx_plugin=(
+  "cabal run plutus-tx-plugin-tests -- --accept"
+  "cabal run size -- --accept"
+)
+tests_plutus_ledger_api=(
+  "cabal run plutus-ledger-api-test -- --accept"
+  "cabal run plutus-ledger-api-plugin-test -- --accept"
+)
+tests_plutus_benchmark=(
+  "cabal run plutus-benchmark-nofib-tests -- --accept"
```

Comment:
How about `cabal run plutus-benchmark --accept`, so we don't need to maintain this list?

Reply:
Good point, I'll change this and other packages as well